edb/graphql/translator.py

```diff
@@ -1383,6 +1383,11 @@ def visit_ObjectFieldNode(self, node):
                 # The binary operator that we need here is "="
                 op = '='
 
+            elif op == 'IN':
+                # Instead of wrapping the values in an array, wrap
+                # them in a set
+                value = qlast.Set(elements=value.elements)
```

Comment:
Currently, `in array_unpack($array)` works better than `in <set>`.

Reply:
Fixed.